python/cudf/cudf/tests/test_setitem.py

```diff
@@ -108,6 +108,12 @@ def test_series_set_item(psr, arg):
     assert_eq(psr, gsr)
 
 
+def test_series_setitem_singleton_range():
+    sr = cudf.Series([1, 2, 3], dtype=np.int64)
+    sr.iloc[:1] = np.asarray([7], dtype=np.int64)
+    assert_eq(sr, cudf.Series([7, 2, 3], dtype=np.int64))
```

Comment:
```suggestion
    sr = cudf.Series([1, 2, 3], dtype=np.int64)
    psr = sr.to_pandas()
    np_arr = np.asarray([7], dtype=np.int64)
    sr.iloc[:1] = np_arr
    psr.iloc[:1] = np_arr
    assert_eq(sr, psr)
```

Reply:
Better to have a pandas comparison in the test?

Reply:
That just asserts we are bug-for-bug compatible with pandas though.